### DISCUSSION TITLE
功能建议：通过网关整合所有的模型到一个docker镜像里面，通过 参数"model": "XXX"来调用不同的LLM，支持对用户token的管理

### DISCUSSION THREAD

**Discussion by lingok1** [LLM-Red-Team/.github#2] [General]
功能建议：通过网关整合所有的模型到一个docker镜像里面，通过 参数"model": "XXX"来调用不同的LLM，支持对用户token的管理

**Comment by lingok1** [comment]
用oneapi转一层有点麻烦，还需要多起用一个服务

**Comment by crapp2** [comment]
每个LLM的token不一样
用one api转一层，方便管理token
